doc/userguide/capture-hardware/ebpf-xdp.rst

```diff
@@ -64,12 +71,17 @@ Kernel
 
 You need to run a kernel 4.13 or newer.
 
-Clang
-~~~~~
+Clang and dependencies
+~~~~~~~~~~~~~~~~~~~~~~
 
 Make sure you have clang (>=3.9) installed on the system  ::
 
- sudo apt-get install clang
+ sudo apt install clang
+
+Some i386 headers will also be needed as eBPF is not x86_64 and some included headers
```

Comment:
What's the reason behind that? i386 is slowly dying as some distributions think about dropping the support.

Reply:
Just a build reason, the used headers are currently these one as it is 'arch independent'. It should evolve with change in libbpf code.